editoast/src/core/stdcm.rs

```diff
@@ -115,28 +114,24 @@ pub struct UndirectedTrackRange {
     pub end: u64,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, ToSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
 #[serde(tag = "status", rename_all = "snake_case")]
 // We accepted the difference of memory size taken by variants
 // Since there is only on success and others are error cases
 #[allow(clippy::large_enum_variant)]
-pub enum STDCMResponse {
+pub enum STDCMCoreResponse {
```

Comment:
Follow up of https://github.com/OpenRailAssociation/osrd/pull/9434#discussion_r1816331474. I still believe that both should be named `STDCMResponse`, one will be named `core::STDCMResponse` and the other one will be `views::timetable::stdcm::STDCMResponse`.